requirements/ci.in

```diff
@@ -0,0 +1,3 @@
+-r base.in
```

Comment:
I don't think `ci.in` should pull in `base.in`. We keep in `ci.in` just the essentials for running the CI, but it should not contain dependencies that are required by the `peek-data` package.